src/components/Dashboard/FirstTimeUser/index.tsx

```diff
@@ -17,9 +17,13 @@ const FirstTimeUser: FC = () => {
   });
 
   const createProfile = useCreateProfile();
-
+  const handleLocalStorage = () => {
+    window.localStorage.setItem('onboardStatus', 'started');
+    window.dispatchEvent(new Event('storage'));
+  };
```

Comment:
There's a localStorageFn.ts file inside utils folder, you can move this in that file.